src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -78,20 +79,20 @@ and the center of the last "effective" echo, sometimes called the "FSL definitio
+|------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| EchoTime               | RECOMMENDED, but REQUIRED if corresponding fieldmap data is present, or the data comes from a multi echo sequence or Arterial Spin Labeling                 | [array][] of [numbers][] | The echo time (TE) for the acquisition, specified in seconds. A single value applies to all volumes, or an array provides a value for each volume. Corresponds to DICOM Tag 0018, 0081 `Echo Time` (please note that the DICOM term is in milliseconds not seconds). 4D files with variable TE should only be used when volume timing is critical for interpretation of the data, as in ASL or variable TE fMRI sequences.                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Apologies for the naive question, but is multi-echo ASL possible? I came across [Paschoal et al. (2019)](https://www.hindawi.com/journals/cmra/2019/5040465/), which seems to compare a dual-echo PCASL sequence to a PCASL sequence with varying echoes.

Reply:
I ask because, if multi-echo ASL is possible, I think it's important that echoes from the same excitation be separated into different files with the `echo` entity, rather than concatenated as they might be in a variable echo sequence.

Reply:
I'll drop this question in the group!

Reply:
@tsalo Hi Taylor, 

Dual-Echo ASL/fMRI does indeed exist but is not used a lot, which is why we chose not to support  this in our current first ASL BIDS version. This specific case would still fit in the current specification, as the first echo would be stored in /perf/ and the second in /func/, for other multi-echo ASL sequences - e.g. to measure the BBB - we will wait with integrating them. The new EchoTime with options both “number or array of numbers” will be helpful for “single-echo” ASL sequences where the echo time varies over volumes (e.g. QUASAR). Hope this clarifies things?

Reply:
Thank you, that is very helpful. Support can wait until the future, but I'm just glad to hear that the plan isn't to concatenate the separate echoes within a single file.